### ISSUE TITLE
Windows Debug Builds are failing

### ISSUE THREAD

**Issue by BergWerkGIS** [mapbox/protozero#37]
e.g.
https://ci.appveyor.com/project/Mapbox/protozero/build/1.0.43/job/oeeb2ail1ue7atft#L289

```
test_cases.obj : error LNK2001: unresolved external symbol __imp___CrtDbgReportW [C:\projects\protozero\tests.vcxproj]
  msvcprtd.lib(locale0_implib.obj) : error LNK2019: unresolved external symbol __imp___free_dbg referenced in function "public: static void __cdecl std::_Fac_node::operator delete(void *)" (??3_Fac_node@std@@SAXPAX@Z) [C:\projects\protozero\tests.vcxproj]
 
  msvcprtd.lib(locale0_implib.obj) : error LNK2019: unresolved external symbol __imp___malloc_dbg referenced in function "public: static void * __cdecl std::_Fac_node::operator new(unsigned int)" (??2_Fac_node@std@@SAPAXI@Z) [C:\projects\protozero\tests.vcxproj]
  C:\projects\protozero\Debug\tests.exe : fatal error LNK1120: 4 unresolved externals [C:\projects\protozero\tests.vcxproj]
```

**Comment by BergWerkGIS** [comment]
/cc @joto @springmeyer 

Building Release and Debug works now https://github.com/mapbox/protozero/commit/7880ff365b59183f311640d9d4257935f37861cc.

**Release `tests.exe`** runs through:
https://ci.appveyor.com/project/Mapbox/protozero/build/1.0.44/job/x9r97wu3jpwmkjg0#L718

But running **Debug `tests.exe`** seem to be stuck on AppVeyor, e.g.:
https://ci.appveyor.com/project/Mapbox/protozero/build/1.0.44/job/secrbvsuim3ns836#L328
because an error message box is displayed.

Occurs with these tests (screenshots from running locally):

![image](https://cloud.githubusercontent.com/assets/4549888/11399608/c2a8b400-9388-11e5-867e-0b4988211bbf.png)


![image](https://cloud.githubusercontent.com/assets/4549888/11399610/c76f468e-9388-11e5-9a5a-ad4313b962d4.png)


![image](https://cloud.githubusercontent.com/assets/4549888/11399612/cc92b5e2-9388-11e5-85ca-68cb8241d830.png)


![image](https://cloud.githubusercontent.com/assets/4549888/11399617/d06c0916-9388-11e5-9955-4bb68c0fd9b7.png)

**Comment by joto** [comment]
Fixed in 9941bac3e6ff9c5094b13aca432d2db282ff6936.

**Comment by BergWerkGIS** [comment]
\o/ great!
